### PR TITLE
Step1 darien

### PR DESCRIPTION
**Pull request by Bear4TW**
Fixed made and pull request resubmitted

Please check my work and provide feedback!
Thank you!

### CODE REVIEW

**Comment by josephum** [comment]
I think you need state_id in the client_vendor table

**Comment by Bear4TW** [comment]
We are waiting for Rumiya to create the State enum so we can uncomment the code. I get error messages if I add state_id to data.sql. What do you think?

File: src/main/resources/data.sql [hunk 1]
```diff
@@ -24,10 +24,10 @@ VALUES (1, '2022-01-05 00:00:00', FALSE, 1, '2022-01-05 00:00:00', 100.00, '1',
        (2, '2022-01-05 00:00:00', FALSE, 1, '2022-01-05 00:00:00', 95.09, '2', false, 'May', '05/12/2022',1),
        (3, '2022-01-05 00:00:00', FALSE, 1, '2022-01-05 00:00:00', 45.87, '3', true, 'June', '06/1/2022',3);
 
-INSERT INTO vendors (created_by, created_time, is_deleted, updated_by, updated_time, name, phone, email, address, state, zip_code, company_type)
-VALUES (1, '2022-01-05 00:00:00', FALSE, 1, '2022-01-05 00:00:00', 'Dalmark', '1234567890', 'dalmark@dalmark.com', '123 Abc St.','VA', '22191', 'CLIENT'),
-       (2, '2022-01-05 00:00:00', FALSE, 1, '2022-01-05 00:00:00', 'S2member', '0987654321', 's2@s2.com', '456 Def St.', 'TX', '78717', 'VENDOR'),
-       (3, '2022-01-05 00:00:00', FALSE, 1, '2022-01-05 00:00:00', 'Tableau', '7894561230', 'tab@tab.com', '789 Ghi St.', 'DC', '22058', 'CLIENT');
+INSERT INTO client_vendor (created_by, created_time, is_deleted, updated_by, updated_time, company_name, phone, email, address, zip_code, enabled)
```
Comment (mgulmira) [inline src/main/resources/data.sql]: probably need to add company_id data

**Comment by josephum** [comment]
Please include state_id commented with todo

https://www.jetbrains.com/help/idea/using-todo.html

File: src/main/resources/data.sql (hunk 1 above)
Comment (Bear4TW) [reply src/main/resources/data.sql]: company_id gives null values but no error messages

**Review by josephum — APPROVED** [review]

**Review by VitalySlabada — APPROVED** [review]
Looks good but could not test, please test before merging

**Pull request merged**